tensorflow/python/ops/parallel_for/math_test.py

```diff
@@ -82,6 +82,11 @@ def test_unary_cwise_complex_ops(self):
     self._test_unary_cwise_ops(complex_ops, True)
 
   def test_unary_cwise_real_ops_1(self):
+    if test.is_built_with_rocm():
```

Comment:
This is not a blocker for this PR, but could you

1) Ideally, add a decorator for skipping such tests similar to https://cs.opensource.google/tensorflow/tensorflow/+/master:tensorflow/python/eager/def_function_xla_jit_test.py;l=134?q=def_function_xla_jit%20file:tensorflow
2) While (1) is not done, use `self.skipTest` instead?

Reply:
@cheshire 

implemented `#2` above. Will file a separate PR to add the decorator requested in `#1` and switch all possible uses of `skipTest(('Fails on ROCm...')` to it